LICENSE.txt

```diff
@@ -0,0 +1,671 @@
+GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
@nedbat, what's your view on this? I'm not totally clear on our licensing policy for libraries.